### ISSUE TITLE
translate Rule Engine DSL to English

### ISSUE THREAD

**Issue by jhkuperus** [scala-rules/scala-rules#4]
The features inside the Rule Engine DSL should preferably be abstracted over and have an English translation implemented as well.

**Comment by NRBPerdijk** [comment]
Has there been any recent progress on this story?

**Comment by jhkuperus** [comment]
I don't think so. Feel free to pickup the branch and translate some stuff. We can look at how it fits V's plans together. (Or maybe he'll pick it up since I heard he was a bit bored) ^_^

**Comment by tculshaw** [comment]
Hi There! 

Currently looking at rules for a scala project I'm working on. What's the approx revised timeline for a release in English? I notice the branch is stale, but obviously a lot of effort had already gone into it?

**Comment by jhkuperus** [comment]
The finance DSL is actually done, I'm going to open a PR for it today. We were revamping the translation of the rules-part of the DSL since we just submitted a CFP-proposal for it.

When would you need it? We'll gladly put it on the top of our list for you :)

**Comment by tculshaw** [comment]
Quick reply:) I'm hoping to put it in as an extension  proposal to a
project I'm working on in the travel industry. Obviously I'd be keen to see
the DSL released, but I can't honestly say I need it yesterday! Happy to
get a rough idea though:)

**Comment by jhkuperus** [comment]
We talked about it today and decided to make this our top priority. Tuesday or Wednesday you'll see a brand new branch pop up for this. (The current branch is too outdated, so we'll be better off cherrypicking what we have onto a new branch). Then we'll try to get it done in the next two or three weeks.

**Comment by tculshaw** [comment]
Fantastic!

**Comment by vzorge** [comment]
I have integrated the changes made into the Translation branch. I now have everything compiling and ready to build further. There are a few new additions to the Dsl that I still need to incorporate into the English translation.

I want to give it a small overhaul before I push these changes. I expect to do a push on saturday the 27th of august, with compiling and working DSL in English. Not sure if I can incorporate all the new features yet, but will do my best.

**Comment by tculshaw** [comment]
On leave next week, but will give it a go on my return. If it goes smoothly I'd be happy to contribute any examples I build.

**Comment by jhkuperus** [comment]
As soon as you've pushed it, we'll run it against our client's project to see if maybe you've broken anything ;)

**Comment by vzorge** [comment]
I have pushed the merge result where everything compiles and all tests are green.

There are some things still left to do.
- Rename some translations, like Determination to Calculation
- Translate DslLoopWordTrait
- Translate DslListFilterWord
- Translate AfrondingsWordsTrait

I will pick them up one by one, starting from the top

**Comment by vzorge** [comment]
I would appreciate a review. I made some choices regarding translations and would like to see what you guys think.

**Comment by NRBPerdijk** [comment]
It's going to sound like a cop-out but for now I don't have any remarks. I think we will need to keep a close look at how the english DSL will actually be used in practice, to see if any of the keywords need adjustment or alternatives...
